client/my-sites/customer-home/cards/features/go-mobile/index.jsx

```diff
@@ -44,8 +44,10 @@ export const GoMobile = ( { email, sendMobileLoginEmail } ) => {
 		<Card className="go-mobile">
 			<div className={ classnames( 'go-mobile__row', { 'has-2-cols': showOnlyOneBadge } ) }>
 				<div className="go-mobile__title">
-					<CardHeading>{ translate( 'Go Mobile' ) }</CardHeading>
-					<h6 className="go-mobile__subheader">{ translate( 'Make updates on the go' ) }</h6>
+					<CardHeading tagName="h2" size={ 16 }>
+						{ translate( 'Go mobile' ) }
+					</CardHeading>
+					<h6 className="go-mobile__subheader">{ translate( 'Make updates on the go.' ) }</h6>
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.